.github/workflows/build-firmware-and-emulator.yml

```diff
@@ -31,14 +31,14 @@ jobs:
             family: osx
             emulator: true
             firmware: false
-            emu_binary: swadge_emulator
+            emu_binary: SwadgeEmulator.zip
```

Comment:
I guess we don't have to differentiate `SwadgeEmulator.zip` and `SwadgeEmulator.zip`?